storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/SnowflakeQueryTemplater.java

```diff
@@ -77,18 +80,15 @@ public SnowflakeQueryTemplater(Map<String, String> databaseConfig, JdbcTemplate
   @Override
   protected List<String> createLoadEntityQuery(String destinationTable, String entitySourceUri) {
     List<String> queries = new ArrayList<>();
-    String csvFormatQuey =
-        String.format(
-            "create or replace file format CSV_format type = 'CSV' field_delimiter = ',' skip_header=1;");
     String copyIntoDestTable =
         String.format(
-            "COPY INTO %s FROM '%s' FILE_FORMAT = CSV_format on_error = 'skip_file' storage_integration = %s;",
-            destinationTable, entitySourceUri, this.storageIntegration);
+            "COPY INTO %s FROM '%s' FILE_FORMAT = %s on_error = 'skip_file' storage_integration = %s;",
+            destinationTable, entitySourceUri, IMPORT_ENTITY_FILE_FORMAT, this.storageIntegration);
     String addRowNum =
         String.format(
-            "CREATE OR REPLACE TABLE %s as SELECT *, ROW_NUMBER() OVER (ORDER BY 1) AS row_number FROM %s;",
+            "CREATE OR REPLACE TEMPORARY TABLE %s as (SELECT *, ROW_NUMBER() OVER (ORDER BY 1) AS row_number FROM %s);",
```

Comment:
When the table is large, we should be able to alter the table by just adding a single row_number column, but not replace the whole table.

Reply:
Can you open a task for this?  We do expect the tables to be large.

Reply:
Okay, I'll create a task in our backlogs